File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.java
class Solution {
    public static int[] productExceptSelf(int[] nums) {
    int[] result = new int[nums.length]; 
    
    for (int i = 0; i < nums.length; i++) {
        int one = 1;
        for (int j = 0; j < nums.length; j++) {
            
            if(i!=j){
            one = one * nums[j]; 
            }
            
        }
        result[i] = one;
        
    }
    return result;
}
    public static void main(String[] args){
        int[] list = {1,2,3,4};
        int[] answer = productExceptSelf(list);
        System.out.println(Arrays.toString(answer));
    }
}  

File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> res(2*nums.size());
        for(int i=0; i<nums.size(); i++){
            int j = nums[i];
            res[i] = res[i+nums.size()] = j;
        }
        return res;
    }
};